### PR TITLE
Feature/windshaft user

### LINKED ISSUE #144: Create DB user for Windshaft
Given read-only permissions, and limit access to tables needed by Windshaft (probably doesn't need access to user info).

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Closes #144.

### CODE REVIEW

File: deployment/ansible/roles/driver.access/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Set table-level access privileges for Windshaft DB role
```
Comment (hectcastro) [inline deployment/ansible/roles/driver.access/tasks/main.yml]: This task needs a `sudo_user: postgres` section to run properly.

File: deployment/ansible/roles/driver.access/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  postgresql_privs: >
+    state=present
+    type=table
+    priv=SELECT
+    objs="{{ windshaft_tables }}"
```
Comment (hectcastro) [inline deployment/ansible/roles/driver.access/tasks/main.yml]: This attribute asks for a comma delimited list, which can be provided with `{{ windshaft_tables | join(",") }}`.

**Comment by hectcastro** [comment]
After the applying the tweaks mentioned above, I got provisioning to complete:

```bash
$ ansible --version
ansible 1.9.3
  configured module search path = None
$ vagrant --version
Vagrant 1.7.4
```

**Comment by flibbertigibbet** [comment]
Provisions for me after upgrading from vagrant 1.7.1 / ansible 1.8.4.

**Comment by flibbertigibbet** [comment]
This should be ready to go. Maybe @kshepard or @moradology would care to take a look.

**Comment by kshepard** [comment]
+1

**Pull request merged**